UM/Settings/InstanceContainer.py

```diff
@@ -40,7 +40,7 @@ class DefinitionNotFoundError(Exception):
 #
 @signalemitter
 class InstanceContainer(QObject, ContainerInterface, PluginObject):
-    Version = 3
+    Version = 4
```

Comment:
Since nothing changed in the storage of these instance containers, I don't see any need to change their version numbers.

Reply:
Replied in https://github.com/Ultimaker/Cura/pull/3663#discussion_r185174128

To trigger a version upgrade so we can move and rename the existing custom profiles, materials, etc. to have their own namespaces (dirs).